src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -1296,7 +1295,7 @@ internal static string CharDescription(char ch)
             while (shift > 0)
             {
                 shift -= 4;
-                sb.Append(Hex[(ch >> shift) & 0xF]);
+                sb.Append(s_hex[(ch >> shift) & 0xF]);
             }
 
             return sb.ToString();
```

Comment:
I assume this CharDescription code does not need to be efficient? (It's not ;)

Reply:
tell me more please :) We loose a lot of performance in RegexCharClass.

Reply:
> We loose a lot of performance in RegexCharClass.

I was referring specifically to these "Description" methods.  They're used in common code paths in release builds?

Reply:
you're right, that method is only used in DEBUG code.